src/controller/command-handlers/authenticate.cpp

```diff
@@ -38,31 +38,30 @@ using namespace electronic_id;
 namespace
 {
 
-// Use common base64-encoding defaults.
-constexpr auto BASE64_OPTIONS = QByteArray::Base64Encoding | QByteArray::KeepTrailingEquals;
```

Comment:
This is intentional as it makes clear what Base64 encoding options we use. If the defaults change in `QByteArray.toBase64()`, we are unaffected. It serves also as documentation/specification, explicit is better than implicit.